msi/release.sh

```diff
@@ -26,9 +64,53 @@ x86_64)
 	die "Unhandled architecture: $ARCH"
 	;;
 esac
-VERSION=$1
-shift
-TARGET="$HOME"/Git-"$VERSION"-"$BITNESS"-bit.msi
+
+for arg
+do
+	case "$1" in
+	-h|--help)
+		showUsage
+		;;
+	-o=*|--outputDir=*)
+		TARGET="${1##*=}"
```

Comment:
It's probably not a big deal, but there should be only one `#` (non-greedy). Otherwise `-o=contains=equal/` would not work as expected.